dynatrace/api/builtin/openpipeline/bizevents/ingestsources/settings/generic_value_assignment.go

```diff
@@ -29,7 +29,7 @@ type GenericValueAssignment struct {
 	Constant           *string                        `json:"constant,omitempty"`           // Constant value
 	Field              *ValueAssignmentFromFieldEntry `json:"field,omitempty"`              // Value from field
 	MultiValueConstant []string                       `json:"multiValueConstant,omitempty"` // Constant multi value
-	Type               AssignmentType                 `json:"type"`                         // Possible Values: `Constant`, `Field`, `MultiValueConstant`
+	Type               AssignmentType                 `json:"type"`                         // Type of value assignment. Possible Values: `constant`, `field`, `multiValueConstant`.
```

Comment:
A consistency question: I noticed that sometimes the comment is prefixed (`Type of value assignment. Possible Values:....`), and other times it just starts with (`Possible Values: ....`). 

I know it's not changing any behavior, but do we want to align this?

Reply:
Team decision: We decided to keep it like that